### PR TITLE
Clean up hash tags

### PR DESCRIPTION
**Pull request by sgarrity**
Make sure hash tags used in URLs aren't used on elements that will scroll the browser
Also open the stats panel if #stats is in the URL

### CODE REVIEW

File: mrburns/main/static/js/main.js [hunk 1]
```diff
@@ -1,10 +1,6 @@
 // vim:set et ts=4 sw=4
 console.log('Calmer than you are.');
 
-$( '.stats-panel-tab' ).click(function() {
-    $( 'body' ).toggleClass( "stats-panel-open" );
-});
-
```
Comment (sgarrity) [inline mrburns/main/static/js/main.js:7]: I just moved this down to line 104.

**Comment by gauthierm** [comment]
This is missing a change in `choice-modal.html` to update the element id.

**Comment by sgarrity** [comment]
> This is missing a change in choice-modal.html to update the element id.

Good catch. Fixed.

**Comment by gauthierm** [comment]
R+ from me.

**Comment by sgarrity** [comment]
Also rebased to address the #video modal changes in PR #19.

**Pull request merged**